packages/core/src/lib/__tests__/code-snippet.spec.svelte

```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
+import CodeSnippet from '$lib/code-snippet.svelte';
+</script>
+
+<CodeSnippet
+  language="json"
+  code={`{ myThing: true }`}
+>
+  This is the caption text.
```

Comment:
Can this be in a named slot? I find it a little confusing that the main content (the code) goes in via a prop and the ancillary content (the caption) goes in via the default slot; it feels backwards in terms of intent

Reply:
Sure